src/pages/api/sepana/utils.ts

```diff
@@ -0,0 +1,250 @@
+import axios from 'axios'
+import {
+  SepanaProject,
+  SepanaProjectJson,
+  SepanaSearchResponse,
+} from 'models/sepana'
+
+const headers = (type: 'read' | 'read/write' | 'admin') => {
+  let key: string
+
+  switch (type) {
+    case 'read':
+      if (!process.env.SEPANA_READ_API_KEY) {
+        throw new Error('Missing SEPANA_READ_API_KEY')
+      }
+      key = process.env.SEPANA_READ_API_KEY
+      break
+    case 'read/write':
+      if (!process.env.SEPANA_READ_WRITE_API_KEY) {
+        throw new Error('Missing SEPANA_READ_WRITE_API_KEY')
+      }
+      key = process.env.SEPANA_READ_WRITE_API_KEY
+      break
+    case 'admin':
+      if (!process.env.SEPANA_ADMIN_API_KEY) {
+        throw new Error('Missing SEPANA_ADMIN_API_KEY')
+      }
+      key = process.env.SEPANA_ADMIN_API_KEY
+      break
+  }
+
+  return {
+    'x-api-key': key,
+    'Content-Type': 'application/json',
+  }
+}
+
+/**
+ * Exhaustively queries all Sepana records.
+ *
+ * @returns Promise containing all project docs from Sepana database
+ */
+export async function queryAllSepanaProjects() {
+  return axios.post<SepanaSearchResponse<SepanaProject>>(
+    `${process.env.NEXT_PUBLIC_SEPANA_API_URL}/search`,
```

Comment:
**suggestion** we can create an axiosInstance that has the `baseUrl` set, which will save having to re-add it every time in these utils.

See https://github.com/jbx-protocol/juice-interface/blob/51384d5ed01e903d49805236c4d3b7e3b28e6d64/src/lib/safe.ts#L14-L17 for example

Reply:
luv